src/parse.jl

```diff
@@ -9,11 +10,21 @@ function parse_header(io)
     seek(io, 10)
     # start, end positions of TEXT, DATA, and ANALYSIS sections
     offsets = Array{Int64}(undef, 6)
+
     for i in 1:6
         # offsets are encoded as ASCII strings
         raw_str = Array{UInt8}(undef, 8)
         read!(io, raw_str)
         offsets_str = String(raw_str)
+
+        # the last two numbers are for the analysis segment
+        # the analysis segment is facultative, although the bytes should
+        # always be there
+        # some cytometers (BD Accuri) do not put the last two bytes
+        # putting "0" bytes in their files is what other cytometers do
```

Comment:
Do you have a citation for this or did you empirically determine this?

Reply:
The standards are from https://isac-net.org/page/Data-Standards (3.1) and https://doi.org/10.1002/(SICI)1097-0320(19970601)28:2%3C118::AID-CYTO3%3E3.0.CO;2-B (3.0), and opening an Accuri file with a text editor shows the missing bytes (replaced by white space)

I'll add refs directly in the comments I guess, although the 3.1 reference doesn't have a permalink unfortunately since it isn't a publication (it should have the same info as the corresponding fcs 3.1 standard paper, https://doi.org/10.1002/cyto.a.20825 , but the latter is more of a text and less of a bullet point list, which makes it harder to follow when checking the binary fcs files)

Reply:
Honestly, you could just add a permalink to your comment above to the code comment and that'll work. Just want a record somewhere so that when i forget why that code is there I have a reference haha